tql_macros/src/analyzer/method.rs

```diff
@@ -29,10 +29,10 @@ pub fn analyze_methods(query: &Query) -> Result<()> {
     let calls = get_method_calls(query);
     let mut errors = vec![];
     for (call, _) in calls {
-        let name = call.method_name.as_ref();
-        if let Some(method) = methods.get(name) {
+        let name = call.method_name.to_string();
```

Comment:
Same here: do we really need to create a string?

Reply:
method_name is `syn::Ident`, which since syn 14.0 is just reexported `proc_macro2::Ident`, which doesn't implement `AsRef<str>`. So `to_string()` is probably the only way to do this, unless i'm missing something